docs/trace/customizing-the-sdk/README.md

```diff
@@ -1,4 +1,4 @@
-# Customizing OpenTelemetry .NET SDK
+# Customizing OpenTelemetry .NET SDK for Tracing
```

Comment:
the docs are under docs/trace folder, so this may not be needed.

Reply:
I poked around some have it, some don't. Metrics does. The reason I added it here is so the link reads better in the SDK README and is consistent with Metrics 🤷